package vista;

import controlador.ConexionBDD;

/*
 * Click nbfs://nbhost/SystemFileSystem/Templates/Licenses/license-default.txt to change this license
 * Click nbfs://nbhost/SystemFileSystem/Templates/Classes/Class.java to edit this template
 */

/**
 *
 * @author Tiririn
 */
public class Main {
       public static void main(String[] args) {
        ConexionBDD bdd=new ConexionBDD();
        bdd.conectar();
    }
 
}
